docs/gamecube.md

```diff
@@ -0,0 +1,48 @@
+---
+section: legacy
+hide:
+  - next
+  - prev
+  - search
+  - tabs
+---
+
+##Gamecube Controller Setup with Official WiiU Adapter
+
+Optional video walkthrough: <https://www.youtube.com/watch?v=4ntXPY4SqNQ>
+
+- Download and install WiiU Adapter Driver: <http://m4sv.com/page/wii-u-gcn-usb-driver>
+- Select **WUP-028** form the drop down in Zadig
+- **DO NOT INSTALL vJoy from here, it's old**
+- Instead download and install the latest vJoy: <https://sourceforge.net/projects/vjoystick/>
```

Comment:
`[latest vJoy](https://sourceforge.net/projects/vjoystick/)`

Reply:
Fixed in https://github.com/smash64-dev/smash64-online/pull/10/commits/3168e0193f3172017cdee399d0f3b9137c64d788